src/components/button/Button.tsx

```diff
@@ -5,128 +5,214 @@
 
 import * as React from 'react';
 import {Loader} from "../loader/Loader";
-// todo: refactor to style components
-import styles from "./button.scss";
+import styled from 'styled-components';
 
+import { loadStyleVariables } from "../../scripts/loadStyleVariables";
 
-var classNames = require('classnames/bind');
-const cx = classNames.bind(styles);
+const styleVariables = loadStyleVariables();
+
+const sizes = {
+  small: '30px',
+  medium: '40px',
+  large: '51px',
+};
+
+const colors = {
+  red: `${styleVariables.red}`,
+  green: `${styleVariables.green}`,
+  blue: `${styleVariables.blue}`,
+};
 
 type Props = {
   id?: string,
   label?: string | React.ReactNode,
   component?: any,
-
   iconLeft?: any,
   iconRight?: any,
   iconOnly?: any,
   outlined?: boolean,
   loading?: boolean,
   percentageDone?: number,
   wide?: boolean,
-  size?: 'small' | 'medium' | 'large';
+  size?: 'small' | 'medium' | 'large',
   rounded?: boolean,
   spaced?: boolean,
   clear?: boolean,
-  color?: string,
+  color?: 'red' | 'green' | 'blue',
```

Comment:
Kunnen we niet beter aan de gebruiker overlaten welke kleur hij wil gebruiken voor de button? Als hij dan een kleur (string) meegeeft die op de color page (moet nog aangemaakt worden) vermeld staat. Hoeft er niet voor elke kleur extra code worden toegevoegd.

Reply:
Dat kan, maar ik dacht dat dit de 3 gebruikte kleuren waren voor een button